src/services/AudioMixer.hxx

```diff
@@ -103,11 +98,11 @@ struct StandardReverbProperties
 
 
 	@param flEchoDelay,AL_ECHO_DELAY,
-	description: delay between the original sound and the first �tap�, or echo instance,range: 0.0 to 0.207
+	description: delay between the original sound and the first �tap�, or echo instance,range: 0.0 to 0.207
 
 
 	@param flEchoLRDelay,AL_ECHO_LRDELAY,
-	description: delay between the first �tap� and the second �tap�.,range: 0.0 to 0.404
+	description: delay between the first �tap� and the second �tap�.,range: 0.0 to 0.404
```

Comment:
Strange characters

Reply:
oh,.. why, I I didn't touch that. shouldn't be a backpack in the first place imho

Reply:
fixed